CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 2.0.0
- - Plugins were updated to follow the new shutdown semantic, this mainly allows Logstash to instruct input plugins to terminate gracefully, 
+ - Plugins were updated to follow the new shutdown semantic, this mainly allows Logstash to instruct input plugins to terminate gracefully,
    instead of using Thread.raise on the plugins' threads. Ref: https://github.com/elastic/logstash/pull/3895
  - Dependency on logstash-core update to 2.0
-
+## 2.0.2
+ - added comments and basic specs
```

Comment:
This should be before 2.0.0 like we do in other changelog see https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/master/CHANGELOG.md